yjit/src/core.rs

```diff
@@ -1733,8 +1774,8 @@ fn branch_stub_hit_body(branch_ptr: *const c_void, target_idx: u32, ec: EcPtr) -
 
     let target_idx: usize = target_idx.as_usize();
     let target = branch.targets[target_idx].as_ref().unwrap();
-    let target_id = target.id;
-    let target_ctx = target.ctx.clone();
+    let target_id = target.get_id();
+    let target_ctx = target.get_ctx().clone();
```

Comment:
Is the clone here necessary since you're already cloning the ctx in `get_ctx()` ?

Reply:
No, thanks for catching it https://github.com/ruby/ruby/pull/7151/commits/8745e9473f8901ff7a088773f84657f4841b9cf3